h2o-algos/src/main/java/hex/schemas/SharedTreeV3.java

```diff
@@ -73,7 +73,7 @@
     @API(help = "Row sample rate per tree (from 0.0 to 1.0)", gridable = true)
     public double sample_rate;
 
-    @API(help = "Row sample rate per tree per class (from 0.0 to 1.0)", level = API.Level.expert, gridable = true)
+    @API(help = "A list/vector of the fraction of rows to sample for each tree, per class (from 0.0 to 1.0)", level = API.Level.expert, gridable = true)
```

Comment:
it's not fraction of rows, but fraction of rows belonging to each class. So I'd say this:

A list of row sample rates per class (relative fraction for each class, from 0.0 to 1.0), for each tree.